ports/espressif/common-hal/socketpool/Socket.c

```diff
@@ -577,3 +599,18 @@ bool common_hal_socketpool_writable(socketpool_socket_obj_t *self) {
     // including returning true in the error case
     return num_triggered != 0;
 }
+
+void socketpool_socket_move(socketpool_socket_obj_t *self, socketpool_socket_obj_t *sock) {
```

Comment:
is the invariant here that the "sock" socket is guaranteed not open, so that it doesn't need to be closed or otherwise operated on?

Reply:
Yes, that should be true. I'll add a check just in case. (Maybe you could get a second websocket request at the same time the first is open.)